docs/core/testing/unit-testing-code-coverage.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Use code coverage for unit testing
+description: Learn how to use the code coverage capabilities for .NET unit tests.
+author: IEvangelist
+ms.author: dapine
+ms.date: 06/15/2020
+---
+
+# Use code coverage for unit testing
+
+Unit tests help to ensure functionality, and provide a means of verification for refactoring efforts. Code coverage is a measurement of the amount of code that is ran as part of a unit test run - either lines, branches, or methods. As an example, if you have a simple application with only two conditional branches of code (_branch a_, and _branch b_), a unit test that verifies conditional _branch a_ will report branch code coverage of 50%.
+
+This article discusses the usage of code coverage for unit testing with coverlet, and it is [open source project on GitHub](https://github.com/coverlet-coverage/coverlet). Coverlet is a cross platform code coverage framework for .NET. Additionally, [coverlet](https://dotnetfoundation.org/projects/coverlet) is part of the .NET foundation.
+
+## Tooling
+
+To use coverlet for code coverage, an existing unit test project must have the appropriate package dependencies, or alternatively rely on [.NET global tooling](../tools/global-tools.md).
```

Comment:
Somewhere it would be good to provide guidance on how you would choose between the three alternatives.

Reply:
That is a great suggestion and I agree. I'll try to add that.